tools/automation/verify/verify_module_load_times.py

```diff
@@ -0,0 +1,92 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+
+from collections import OrderedDict
+import re
+from subprocess import check_output, STDOUT, CalledProcessError
+import sys
+
+def init(root):
+    parser = root.add_parser('module-load-perf', help='Verify that modules load within an acceptable timeframe.')
+    parser.set_defaults(func=run_verifications)
+
+
+def mean(data):
+    """Return the sample arithmetic mean of data."""
+    n = len(data)
+    if n < 1:
+        raise ValueError('len < 1')
+    return sum(data)/float(n)
+
+
+def sq_deviation(data):
+    """Return sum of square deviations of sequence data."""
+    c = mean(data)
+    return sum((x-c)**2 for x in data)
+
+
+def pstdev(data):
+    """Calculates the population standard deviation."""
+    n = len(data)
+    if n < 2:
+        raise ValueError('len < 2')
+    ss = sq_deviation(data)
+    return (ss/n) ** 0.5
+
+def run_verifications(args):
+    regex = r"[^']*'([^']*)'[\D]*([\d\.]*)"
+    num_runs = 3
+
+    high_threshold = 25
+    high_modules = ['network', 'vm', 'batch']
```

Comment:
Why this list?

Reply:
Good point. I thought that was a style thing from the non-Python world that was discouraged in Python, but I just found it in PEP8 :grin: I'll change it.

But, as to why it is there, it will likely change, but when most modules load in under 10, I didn't want to set a single threshold that would be much higher based on a few of the more complex modules. I'd rather apply the low threshold to everything, and then, if there's a compelling reason, grant a higher threshold on a case-by-case basis.